cirq-core/cirq/protocols/__init__.py

```diff
@@ -88,6 +88,7 @@
     to_json,
     read_json,
     obj_to_dict_helper,
+    dataclass_json_dict,
```

Comment:
I was going to say sort imports, but it looks like we haven't been doing that consistently. Really needs to be automated...

Reply:
Craig and I are in solidarity that we won't manually sort imports